File: Springboot Code/AgroRent/src/main/java/com/majorproj/agrorent/dto/BookingResponseOwnerDTO.java
package com.majorproj.agrorent.dto;

import java.time.LocalDate;

import com.majorproj.agrorent.entities.BookingStatus;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class BookingResponseOwnerDTO {
    private Long id;
    private LocalDate startDate;
    private LocalDate endDate;
    private BookingStatus status;
    private double totalAmount;
    private boolean isPaid;
    private String renterName;
    private String eqipmentName;
}

File: Dot Net Code/AgroRent/DTOs/BookingResponseOwnerDTO.cs
namespace AgroRent.DTOs
{
    public class BookingResponseOwnerDTO
    {
        public int Id { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Status { get; set; } = string.Empty;
        public double TotalAmount { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime UpdatedOn { get; set; }
        public int EquipmentId { get; set; }
        public string EquipmentName { get; set; } = string.Empty;
        public int RenterId { get; set; }
        public string RenterName { get; set; } = string.Empty;
        public string RenterEmail { get; set; } = string.Empty;
    }
}
